boards/arm/rddrone_fmuk66/rddrone_fmuk66_defconfig

```diff
@@ -13,3 +14,9 @@ CONFIG_OSC_EXTERNAL=y
 # Enable MPU
 CONFIG_ARM_MPU=y
 CONFIG_HW_STACK_PROTECTION=y
+
+# Enable Regulators
+CONFIG_REGULATOR=y
+CONFIG_REGULATOR_FIXED=y
+CONFIG_REGULATOR_FIXED_INIT_PRIORITY=75
+CONFIG_REGULATOR_LOG_LEVEL_WRN=y
```

Comment:
please stick to minimal options in board files, per porting guidelines. enabling log levels here is wrong

Reply:
As per your suggestion, I have removed - CONFIG_REGULATOR_LOG_LEVEL_WRN
But rest is necessary for most of the sensors on this board to work.
So, I would request you to accept the others.